app/controllers/events_controller.rb

```diff
@@ -0,0 +1,37 @@
+class EventsController < ApplicationController
+  before_action :require_user, except: [:index]
+  def index
+    @past = Event.past
+    @upcoming = Event.upcoming
+  end
+
+  def new
+    @event = current_user.events.build
+  end
+
+  def create
+    @event = current_user.events.build(event_params)
+
+    if @event.save
+      flash[:notice] = 'Event has been created'
+      redirect_to events_path
+    else
+      flash[:danger] = 'Something wrong'
+      render 'new'
+    end
+  end
+
+  def show
+    @event = Event.find(params[:id])
+    # @users = User.all
+    @event.attendees.each do |e|
+      @users = User.where.not(name: e.name).where.not(name: @event.creator.name)
```

Comment:
- [ ] This line has too much application logic for a controller. Refactor it into a model method.
ie: `@users = User.example_method(e, @event)`

Reply:
Done with changes